src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.model;
```

Comment:
It looks like you removed completely `Operation` class but not moved it to it's file